Naval Act Changes

Ruling-0 opened the pull request:
- USA Naval Act idea
  - No longer costs political power
  - No longer gives 20% refit speed, 15% dock output, -50% module upgrade xp cost
  - Dock construction bonus increased from 20% to 50%
  - Naval base construction bonus increased from 20% to 25%
  - Consumer goods cost reduced from 3% to 2%
- Two Ocean Navy Act now gives the Modernize the Fleet refit idea for 1 year

Changes give USA the refit bonus every other naval major has, and retune Naval Act of 1936 to function as an early way to build dockyards (removing unnecessary output+refit, increasing dock construction speed to compensate for elongated London Naval Treaty)

snyperfire93 reviewed (approved):
Definitely think that will help USA players

Ruling-0 commented:
Rebalanced a bit and included a bug fix. Dropped LNT from -25% to -15% dock construction, dropped the US idea from +50% to +35%, fixed JAP/USA sub doctrine focuses (JAP only worked for Fleet in Being, USA was still tech bonus not doctrine cost reduction). Updated changelog:

- London Naval Treaty dockyard construction penalty reduced from -25% to -15%
- USA Naval Act idea
  - No longer costs political power
  - No longer gives 20% refit speed, 15% dock output, -50% module upgrade xp cost
  - Dockyard construction bonus increased from 20% to 35%
  - Naval base construction bonus increased from 20% to 25%
  - Consumer goods cost reduced from 3% to 2%
- Two Ocean Navy Act now gives the Modernize the Fleet refit idea for 1 year
